sdk/node/Libplanet.Node/DataAnnotations/AppProtocolVersionAttribute.cs

```diff
@@ -0,0 +1,33 @@
+#pragma warning disable SA1402 // File may only contain a single type
+using System.ComponentModel.DataAnnotations;
+
+namespace Libplanet.Node.DataAnnotations;
+
+[AttributeUsage(AttributeTargets.Property)]
+public sealed class AppProtocolVersionAttribute : RegularExpressionAttribute
+{
+    public const string VersionPattern = @"\d+";
+    public const string AddressPattern = @"[0-9a-fA-F]{40}";
+    public const string Base64Pattern
+        = @"(?:[A-Za-z0-9+\.]{4})*(?:[A-Za-z0-9+\.]{2}==|[A-Za-z0-9+\.]{3}=)?";
```

Comment:
Is it needed when adding each option?

Reply:
> Is it needed when adding each option?

This is required for validation of APV values in appsettings.json. Does this element pose a significant security risk?

Reply:
It feels very chore to define the pattern as a regex every time we add an option. Is there a better way? Or is it because of the special of APV that we should apply the regex pattern only for APV options?

Reply:
It is not mandatory to define a regex pattern each time you add an option, it is just to ensure that the appropriate values are used as you finalise your `appsettings.json`.